pkg/occlient/occlient_test.go

```diff
@@ -717,13 +721,15 @@ func TestParseImageName(t *testing.T) {
 
 	tests := []struct {
 		arg     string
+		want0   string
```

Comment:
`want0` looks odd. Can we make it `want4`, or change the order.

Reply:
Ack